src/css/main.less

```diff
@@ -1307,27 +1304,56 @@ fieldset[disabled] .btn-info.active {
       height: @base-spacing-unit*2;
       margin: 0 0 0 @base-spacing-unit;
       width: @base-spacing-unit*2;
+      vertical-align: baseline;
     }
 
     &:last-child {
-      &::after {
-        opacity: 0;
-      }
-
       a {
         color: darken(@link-color, 40%);
         cursor: default;
+
+        &::before {
+          content: "";
+        }
+      }
+
+      &::after {
+        opacity: 0;
       }
     }
 
     a {
       color: @link-color;
+      white-space: nowrap;
+      overflow: hidden;
 
       &:hover {
         color: darken(@link-color, 40%);
       }
     }
   }
+
+  &.collapsed {
+    a {
+      width: 0.75em;
+
+      &::before {
```

Comment:
Could you add a ```display: inline-block``` here.
This solves the problem with the bluelines in FF:
![visuals](https://cloud.githubusercontent.com/assets/859154/10581155/8a9ff9a2-7681-11e5-9037-692b2c06b489.png)

But needs some afterworks with the positions:
![b-without](https://cloud.githubusercontent.com/assets/859154/10581170/94b1d104-7681-11e5-8d1b-9ded42b2cfdc.png)